### PR TITLE
bib-tool 2.63

### PR DESCRIPTION
**Pull request by adamliter**
This updates the `bib-tool` formula from version 2.61 to version 2.63.

### CODE REVIEW

File: Library/Formula/bib-tool.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class BibTool < Formula
   desc "Manipulates BibTeX databases"
   homepage "http://www.gerd-neugebauer.de/software/TeX/BibTool/index.en.html"
-  url "https://github.com/ge-ne/bibtool/releases/download/BibTool_2_61/BibTool-2.61.tar.gz"
-  sha256 "8eaf351f1685078345a4446346559698fb58d8d1dfdf057418e5221132f9a8a4"
+  url "https://github.com/ge-ne/bibtool/archive/BibTool_2_63.tar.gz"
```
Comment (DomT4) [inline Library/Formula/bib-tool.rb]: Any idea why 2.61 is still listed as the Latest Release on their GitHub tags?

Comment (adamliter) [inline Library/Formula/bib-tool.rb]: @DomT4 That's a good question. I'm honestly not sure, but the latest version at the homepage is listed as 2.63.

Comment (DomT4) [inline Library/Formula/bib-tool.rb]: Good enough for me, Thanks!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/577cdcdba2460e0bf10be026519046da55c77e97. Thank you for another contribution to Homebrew @adamliter! :smiley_cat:

**Comment by adamliter** [comment]
Thanks, @DomT4! :tada:

**Pull request closed without merging**